_pages/about/team.html

```diff
@@ -39,20 +39,24 @@ <h3>About us</h3>
             <h4 class="text-left animated hiding appear-animation-trigger" data-animation="fadeInUp" data-delay="0">
                 Local Committee
             </h4>
-            {% for teamMember in site.data.team-loc %}{% include team-member.html teamMember=teamMember %}{% endfor %}
+            <p class="text-center">
+            <a href="{% link _pages/about/team-loc.html %}" class="btn btn-primary waves-effect waves-button waves-light waves-float">View the Local committee</a>
```

Comment:
I would leave the LOC committee on this page instead of creating a new page for it.
No worries to have program committee and academic committee on separate pages (though I'm not sure it's needed).

Reply:
@jbelien I edited the page with a new design. I left the local team there, but added three button links on top for each committee and also to the other committees se that there is a way to move between the pages of different committees.

Reply:
@jbelien since I made many changes to this MR, I am going to merge it. And I can make any new improvements on a new one as the team has been asking for some of the changes already.